src/bridge/cockpitfsread.c

```diff
@@ -192,7 +192,9 @@ static void
 cockpit_fsread_prepare (CockpitChannel *channel)
 {
   CockpitFsread *self = COCKPIT_FSREAD (channel);
+  const gchar *binary;
```

Comment:
please move declarations closer to site of use, and use `g_autoptr()` if possible